lib/data_update_scripts/20210722135549_profile_website_url_format.rb

```diff
@@ -0,0 +1,29 @@
+module DataUpdateScripts
+  class ProfileWebsiteUrlFormat
+    def run
+      profiles_to_fix.each do |profile|
+        fix_or_clear_website_url(profile)
+        profile.save || log_failure(profile)
+      end
+    end
+
+    def fix_or_clear_website_url(profile)
+      profile.website_url =
+        begin
+          new_url = "https://#{profile.website_url}"
+          uri = URI.parse(new_url)
+          uri.scheme.present? && uri.host.present? && new_url || ""
+        end
+    end
+
+    def log_failure(profile)
+      Rails.logger.warn("Attempted to update website_url for profile #{profile.id} but failed with #{profile.errors}")
+    end
+
+    def profiles_to_fix
+      Profile
+        .where.not("website_url like 'http__/%'")
```

Comment:
originally I was going to use `http%`, but decided http, any two characters, and a slash, would match https:// http:// and very little else (the main focus here is to fixup websites without any scheme, rather than ones with tiny typos in the scheme). 

It's still possible this ends up clearing a "human-obvious" typo of the form https;// or something, but that was already broken (the generated link would have been a problem or raised an error).